airbyte-integrations/connectors-performance/source-harness/build.gradle

```diff
@@ -14,8 +14,6 @@ application {
     applicationDefaultJvmArgs = ['-XX:+ExitOnOutOfMemoryError', '-XX:MaxRAMPercentage=75.0']
 }
 dependencies {
-    implementation project(':airbyte-db:db-lib')
-    implementation project(':airbyte-integrations:bases:base-java')
```

Comment:
Shouldn't this build.gradle be similar to that in `destination-harness`? That one applies the `airbyte-java-connector` plugin and this one doesn't.

Reply:
Good catch! I actually added `airbyte-java-connector` to `destination-harness` because it had a mapped dependency on on `db-lib` and `base-java`. I didn't like adding the `airbyte-java-connector` dependency, because these harnesses aren't connectors themselves.

Upon further testing, I found that both projects compile fine without db-lib and base-java, and without the mapping of the CDK.

So, I've removed `airbyte-java-connector` plugin from `destination-harness` and confirmed that both of these still work:

```console
./gradlew :airbyte-integrations:connectors-performance:source-harness:build
./gradlew :airbyte-integrations:connectors-performance:destination-harness:build
```